app/src/main/java/com/sdp/movemeet/view/navigation/Navigation.java

```diff
@@ -182,7 +184,7 @@ public void createDrawer() {
         navigationView.setNavigationItemSelectedListener(this::onNavigationItemSelected);
         navigationView.setCheckedItem(this.activityId);
 
-        fillNavigationProfileFields(new TextView[] {fullName, email, phone});
+        if (profileField) fillNavigationProfileFields(new TextView[] {fullName, email, phone});
```

Comment:
In which cases could the `profileField` variable suddenly be set to false?

Reply:
It is a variable identical to enablenav. It can be set to false in tests for example if we don't want to try to get the user's information when creating the menu.

Reply:
Ok ok I see, good trick!